src/main/java/staffconnect/model/Model.java

```diff
@@ -84,4 +85,10 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
+
+    /**
+     * Updates the sort attribute of the sorted person list to sort by the given {@code predicate}.
+     * @throws NullPointerException if {@code predicate} is null.
```

Comment:
should these be `{@code comparator}` instead?

Reply:
FISED!